esptool.py

```diff
@@ -435,6 +435,34 @@ def _setRTS(self, state):
         # request is sent with the updated RTS state and the same DTR state
         self._port.setDTR(self._port.dtr)
 
+    def _sendBreak(self):
```

Comment:
This private function isn't used any more, can be removed.

Reply:
Actually, I need it in the one of reset methodologies and instead of digging through the innards of the esptool, i though it would be handier to just have this small function here. I can make it non-private though, if that helps, but in that case I would suggest the RTS and DTR function to become non-private too.

Reply:
Suggest it makes sense for `setDTR()`, `setRTS()`, and `port` to be "private" (for Python's permissive definition of "private"), because the ESPROM class expects to manage the state of these things itself rather than have that state manipulated by a user of the ESPROM object.

If subclassing the ESPROM class (or patching) then you can still call these in the subclass (or the patch), with the implicit association that you're messing with the "internals" here and the parent class might also change this state sometimes.

Suggest it's also correct to add a function like `_sendBreak()` in that same external place, where it is needed and will be used.